Game 2.js

```diff
@@ -0,0 +1,76 @@
+const Board = require("./Board.js")
+const HumanPlayer = require("./HumanPlayer.js")
```

Comment:
Since you're not using the humanplayer and computerplayer on this class, you can delete these two imports